services/gateway/tests/constants/generateDocs.js

```diff
@@ -584,6 +584,12 @@ const createApiDocsExpectedResponse = {
 						$ref: '#/definitions/badRequest',
 					},
 				},
+				503: {
+					description: 'Service Unavailable',
+					schema: {
+						$ref: '#/definitions/serviceUnavailable',
+					},
+				},
```

Comment:
would this api return 503 as this is only dependant on gateway?

Reply:
Updated the code, API used to timeout incase node was not reachable. Now should throw 503.